File: Chapter 06/Python Code/_6_4.py
x=input("Enter first sign of the game ")
y=input("Enter second sign of the game ")
if (x== "scissor" and y== "paper") or (x== "stone" and y== "scissor") or (x== "paper" and y== "stone"):
    print( "x is the winner")
if (y== "scissor" and x== "paper") or (y== "stone" and x== "scissor") or (y== "paper" and x== "stone") :
    print("y is the winner")
if (x== "scissor" and y== "scissor") or (x== "stone" and y== "stone") or (x== "paper" and y== "paper"):
    print( "There is a tie")

File: Chapter 06/C Code/_6_4.c
#include<stdio.h>
void main()
{
char x,y;
fflush(stdin);
printf("Enter signs of the game: s for scissor, p for paper, r for stone: ");
scanf("%c", &x);
fflush(stdin);
scanf("%c", &y);

if ((x== 's' && y== 'p') || (x== 'r' && y== 's') || (x== 'p' && y== 'r'))
		printf( "x is the winner");
if ((y== 's' && x== 'p') || (y== 'r' && x== 's') || (y== 'p' && x== 'r'))
		printf("y is the winner");
if ((x== 's' && y== 's') || (x== 'r' && y== 'r') || (x== 'p' && y== 'p'))
		printf( "There is a tie");
}

